index.html

```diff
@@ -0,0 +1,259 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <!-- NAVBAR -->
+    <nav class="navbar">
+        <div class="navbar-innercontainer">
+            <span class="nav-logo">LifeSoft</span>
+            <ul class="nav-items">
+                <li class="nav-link">
+                    <a href="#" class="link active">Portfolio</a>
+                </li>
+                <li class="nav-link"><a href="#" class="link">About</a></li>
+                <li class="nav-link"><a href="#" class="link">Contact</a></li>
+                <li class="nav-link">
+                    <a href="#"><img src="./Assets/logos/envelope.png" class="icon" alt="" /></a>
+                </li>
+            </ul>
+            <div class="baricon">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </div>
+    </nav>
+    <section class="herosection">
+        <div class="textcontainer">
+            <h1 class="h5">Hey There. I’m Life.</h1>
+            <h6 class="h6">I'm a Software Developer</h6>
+            <p class="hero-paragraph">
+                I can help you build a product , feature or website Look through some
+                of my work and experience! If you like what you see and have a project
+                you need coded, don’t hesitate to contact me.
+            </p>
+        </div>
+
+        <ul class="icon-container">
+            <li class="icon_link">
+                <a href="#"><img class="icon-image" src="./Assets/logos/Vector.png" alt="" /></a>
+            </li>
+            <li class="icon_link">
+                <a href="#"><img class="icon-image" src="./Assets/logos/linkin.png" alt="" /></a>
+            </li>
+            <li class="icon_link">
+                <a href="#"><img class="icon-image" src="./Assets/logos/fingers.png" alt="" /></a>
+            </li>
+            <li class="icon_link">
+                <a href="#"><img class="icon-image" src="./Assets/logos/twitter.png" alt="" /></a>
+            </li>
+            <li class="icon_link">
+                <a href="#"><img class="icon-image" src="./Assets/logos/letterM.png" alt="" /></a>
+            </li>
+        </ul>
+    </section>
+
+    <!-- MIDDLE SECTION -->
+    <div class="works">
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/Peter-Dumbari/Portofolio-Microverse/pull/2#discussion_r1148404452) please consider using a `section` tag for wrapping all project cards instead of `div` 💪